.gitignore

```diff
@@ -0,0 +1,17 @@
+/.bundle/
+/.yardoc
+Gemfile.lock
+/_yardoc/
+/coverage/
+/doc/
+/pkg/
+/spec/reports/
+vendor
+*.gem
+
+# rspec failure tracking
+.rspec_status
+
+# C++ stuff
+*.bin
+*.bin.dSYM
```

Comment:
Add `.DS_Store` (unless you can explain what it is and why it should be committed). Also, Git likes to have an endline a the end of a file.